process/elasticproc/logsevents/nftsProcessor_test.go

```diff
@@ -50,7 +50,8 @@ func TestNftsProcessor_processLogAndEventsNFTs(t *testing.T) {
 		Issuer:     "",
 		Nonce:      uint64(19),
 		Data: &data.TokenMetaData{
-			Creator: hex.EncodeToString([]byte("creator")),
+			Creator:    hex.EncodeToString([]byte("creator")),
+			Attributes: []byte{},
```

Comment:
`make([]byte, 0)` instead of literal declaration
This test (and also the other tests) should have worked with nil field. You can test them

Reply:
done